packages/backend/migrations/20220228162503_create_incoming_payments_table.js

```diff
@@ -12,8 +12,6 @@ exports.up = function (knex) {
     table.string('externalRef').nullable()
     table.uuid('connectionId').notNullable()
 
-    table.uuid('assetId').notNullable()
-    table.foreign('assetId').references('assets.id')
```

Comment:
:confounded: I forgot about the _possibility_ of a payment pointer's asset changing, and thus the need to store the asset on the payment...